packages/backend/src/accounting/tigerbeetle/transfers.ts

```diff
@@ -62,10 +62,11 @@ export async function createTransfers(
       id: 0n,
       debit_account_id: 0n,
       credit_account_id: 0n,
-      user_data: 0n,
-      reserved: 0n,
```

Comment:
Is `reserved` no longer required here?

Reply:
No, it was dropped, see [here](https://docs.tigerbeetle.com/reference/operations/create_transfers). There is only a `reserved_flag` now, but if I read that correctly, that defaults to 0.